interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: When calling a super, we ask that our new thing we're making, have an similiar outline as a standard feature.
 
-Researched answer:
+Researched answer: Keyword; calls on a parent class with the same name as the method thatb calls super...(ref.rubyguides.com) 
```

Comment:
A lot of these researched answers are almost identical to your sources.  Please make sure your answers are **_formed in your own words_**.